### PR TITLE
Make owner of files/directories configurable

### PR DESCRIPTION
**Pull request by dhoppe**

### CODE REVIEW

File: manifests/broker/config.pp [hunk 1]
```diff
@@ -14,6 +14,7 @@
   Boolean $service_restart         = $kafka::broker::service_restart,
   Hash $config                     = $kafka::broker::config,
   Stdlib::Filemode $config_mode    = $kafka::broker::config_mode,
+  String $user                     = $kafka::broker::user,
```
Comment (bastelfreak) [inline manifests/broker/config.pp:17]:
```suggestion
  String[1] $user                  = $kafka::broker::user,
```

File: manifests/mirror/config.pp [hunk 2]
```diff
@@ -15,6 +15,7 @@
   Hash $consumer_config            = $kafka::mirror::consumer_config,
   Hash $producer_config            = $kafka::mirror::producer_config,
   Stdlib::Filemode $config_mode    = $kafka::mirror::config_mode,
+  String $user                     = $kafka::mirror::user,
```
Comment (bastelfreak) [inline manifests/mirror/config.pp:18]:
```suggestion
  String[1] $user                  = $kafka::mirror::user,
```

File: manifests/producer/config.pp [hunk 3]
```diff
@@ -14,6 +14,7 @@
   Boolean $service_restart         = $kafka::producer::service_restart,
   Hash $config                     = $kafka::producer::config,
   Stdlib::Filemode $config_mode    = $kafka::producer::config_mode,
+  String $user                     = $kafka::producer::user,
```
Comment (bastelfreak) [inline manifests/producer/config.pp:17]:
```suggestion
  String[1] $user                  = $kafka::producer::user,
```

**Comment by dhoppe** [comment]
@bastelfreak Well, you missed some classes. However these classes should be private and the type checking should be done at ```manifests/broker.pp```, for example.

**Comment by bastelfreak** [comment]
rebased and approved it

**Pull request merged**